doc/api/vm.md

```diff
@@ -201,8 +201,7 @@ added: v0.3.1
 
 * `sandbox` {Object}
 
-If given a `sandbox` object, the `vm.createContext()` method will [prepare
-that sandbox][#vm_what_does_it_mean_to_contextify_an_object] so that it can be
+If given a `sandbox` object, the `vm.createContext()` method will [prepare that sandbox][] so that it can be
```

Comment:
Can you wrap after the `will`, so it fits on the 80 character limit? You can re-wrap the following lines.